### ISSUE TITLE
bad performace in robotwin evaluation

### ISSUE THREAD

**Issue by justimyhxu** [thu-ml/Motus#11]
Hi, thanks for the excellent work and for releasing the codebase!

I followed the README instructions to set up the evaluation environment for **RobotWin** with **Motus**, and tested the rollout trajectories on the task **“put the object in the cabinet.”** However, I observed a noticeable discrepancy between the reported and reproduced performance.

**Test results:**
- Reproduced success rate: ~**45%**
- Reported in the paper (supplementary):
  - ~**88%** in the *clean* setting  
  - ~**71%** in the *randomized* setting

I would like to ask:
1. Are there any additional evaluation details (e.g., environment version, random seed handling, number of rollouts, or success criteria) that might affect this result?
2. Were the reported numbers averaged over multiple runs or random seeds?
3. Have you tested the reproducibility of this task multiple times on a fresh environment to confirm the reported performance?

I would really appreciate any guidance on what might cause this gap, or pointers to specific evaluation settings to double-check. I’m happy to provide more details (logs, configs, seeds) if helpful.

Thanks again for sharing such a great project!

<img width="1842" height="1554" alt="Image" src="https://github.com/user-attachments/assets/c7803a8b-bd2a-45a5-9c92-400046867d0b" />

**Comment by HongzheBi** [comment]
In addition, are the success metrics of other tasks so different? You can calculate if there is a severe drop in the overall average success rate for the 50 tasks.

**Comment by strive789** [comment]
Have you solved this issue? I can reproduce the reported scores using the open-source weights. However, when I train on RoboTwin with the same settings as in the paper—batch size 32×8, 40,000 steps—under three settings (no pretrain, loading Stage-1 weights, loading Stage-2 weights), the success rate in the Random setting is all below 60%. Did you encounter this problem as well?
@HongzheBi @hanyangyu1021 @justimyhxu

**Comment by HuailiangMa** [comment]
> Have you solved this issue? I can reproduce the reported scores using the open-source weights. However, when I train on RoboTwin with the same settings as in the paper—batch size 32×8, 40,000 steps—under three settings (no pretrain, loading Stage-1 weights, loading Stage-2 weights), the success rate in the Random setting is all below 60%. Did you encounter this problem as well? [@HongzheBi](https://github.com/HongzheBi) [@hanyangyu1021](https://github.com/hanyangyu1021) [@justimyhxu](https://github.com/justimyhxu)

我也遇到了这个问题，用Robotwin生成的550条数据做单任务后训练，训练15000步之后模型效果跟论文报告有很大差别，理论上多任务训练40000步，我这个训练15000步单任务应该可以，但实际上单任务成功率只有20%，远不如报告里的88%

**Comment by Songyao-Jin** [comment]
I also have this problem. When I finetune it for 40k step, the performance is much lower than the reported result.
My total batch size is 256 with 40k training steps.
